clusters/publick8s.yaml

```diff
@@ -0,0 +1,19 @@
+helmDefaults:
+  atomic: true
+  force: false
+  timeout: 300
+  wait: true
+repositories:
+  # https://github.com/DataDog/helm-charts/
+  - name: datadog
+    url: https://helm.datadoghq.com
+releases:
+  - name: datadog
+    namespace: datadog
+    chart: datadog/datadog
+    version: 3.10.0
+    values:
+      - "../config/ext_datadog.yaml.gotmpl"
+      - "../config/ext_datadog_publick8s.yaml"
```

Comment:
[Don't forget to update the clustername in `config/ext_datadog_publick8s.yaml` (key `datadog.clusterName`)](https://github.com/lemeurherve/kubernetes-management/blob/4b6b20d84e353fa4177519cc185e49565e4fcd15/config/ext_datadog_publick8s.yaml#L2) => name was already `publick8s` nice!